docs/commands/push.md

```diff
@@ -1,93 +1,186 @@
 # `push`
 
+## Overview
+
 Redocly Workflows integrates with [popular version control services](https://redoc.ly/docs/workflows/sources/) and uses them as the source of your API definitions to help you automatically validate, build, and deploy API reference docs and developer portals. This approach requires you to give Redocly Workflows access to your repositories.
 
-As an alternative, you can use the OpenAPI CLI `push` command and set up your own CI pipeline for updating API definitions without granting Redocly Workflows access to your repositories. This way, you can control the frequency of API definition updates and still have the benefit of using Redocly Workflows to preview documentation and portal builds, and manage versions in the API registry.
+As an alternative, you can use the OpenAPI CLI `push` command and set up your own CI pipeline to update API definitions without granting Redocly Workflows access to your repositories. This way, you can:
+
+- Control the frequency of API definition updates.
+- Benefit from using Redocly Workflows to preview documentation and portal builds.
+- Manage versions in the API registry.
 
-Apart from uploading your API definition file, the `push` command can automatically upload other files if they are detected or referenced in the API definition. More specifically, the command can upload:
+Apart from uploading your API definition file, the `push` command can automatically upload other files if they are detected or referenced in the API definition:
 
 - the `.redocly.yaml` configuration file
+- the `package.json` file (if it exists) from the folder where you're executing the `push` command. Redocly Workflows will use the `@redocly/openapi-cli` version specified in `package.json`.
 - the HTML template and the full contents of the folder specified as the `referenceDocs > htmlTemplate` parameter in `.redocly.yaml`.
 
+:::warning
+If a plugin is referenced in the `.redocly.yaml` file, the `push` command will recursively scan the folder containing the plugin and upload all `.js`, `.json`, `.mjs` and `.ts` files.
+
+Make sure that each plugin has all the required files in its folder, otherwise they will not be uploaded.
+:::
+
+By default, the `push` command only updates an existing API definition version. If an API with the provided name and version doesn't exist in your organization, it will not be created automatically, and the command will exit with an error exit code.
+
```

Comment:
I think there may actually be a bug, because it should also have an exit code if there is an error... and it might not.

https://www.redhat.com/sysadmin/exit-codes-demystified#:~:text=When%20you%20execute%20a%20command,from%20your%20command%20or%20script.&text=Programmers%20use%20exit%20codes%20to%20help%20debug%20their%20code.